policyengine_us/reforms/ctc/eppc/expanded_ctc.py

```diff
@@ -0,0 +1,423 @@
+            school_meals_daily_subsidy = nslp_per_child + sbp_per_child
+            daily_paid_subsidy = tax_unit.spm_unit(
+                "school_meal_paid_daily_subsidy", period
+            )
+            net_daily_subsidy_per_child = (
+                school_meals_daily_subsidy - daily_paid_subsidy
+            )
+            p_school_meals = parameters(period).gov.usda.school_meals
+            children = add(tax_unit, period, ["is_in_k12_school"])
+            school_meal_max_value = (
+                net_daily_subsidy_per_child
+                * children
+                * p_school_meals.school_days
+            )
+            # Get TANF grant standards (maximum amounts)
+            max_federal_tanf = tax_unit.spm_unit("tanf_max_amount", period)
+            max_ny_tanf = tax_unit.spm_unit("ny_tanf_grant_standard", period)
+            tanf_dem_eligible = tax_unit.spm_unit(
+                "is_demographic_tanf_eligible", period
+            )
+            max_dc_tanf = tax_unit.spm_unit("dc_tanf_grant_standard", period)
+            max_co_tanf = tax_unit.spm_unit("co_tanf_grant_standard", period)
+            max_tanf = (
+                max_co_tanf + max_dc_tanf + max_federal_tanf + max_ny_tanf
+            ) * tanf_dem_eligible
+            return snap_max_allotment + school_meal_max_value + max_tanf
+
+    # At the core of the reform, we want to coampre the tax liability of two individuals
+    # filing as single people vs filing as a couple with the expanded CTC
+
+    class basic_standard_deduction_indiv(Variable):
+        value_type = float
+        entity = Person
+        label = "Basic standard deduction"
+        definition_period = YEAR
+        unit = USD
+        reference = "https://www.law.cornell.edu/uscode/text/26/63#c_2"
+
+        def formula(person, period, parameters):
+            std = parameters(period).gov.irs.deductions.standard
+            separate_filer_itemizes = person.tax_unit(
+                "separate_filer_itemizes", period
+            )
+            dependent_elsewhere = person.tax_unit(
+                "head_is_dependent_elsewhere", period
+            )
+
+            # Calculate secondary earner deduction using single filing status
+            deduction_amount = std.amount["SINGLE"]
+
+            standard_deduction_if_dependent = min_(
+                deduction_amount,
+                max_(
+                    std.dependent.additional_earned_income
+                    + person.tax_unit("tax_unit_earned_income", period),
+                    std.dependent.amount,
+                ),
+            )
+
+            return select(
+                [
+                    separate_filer_itemizes,
+                    dependent_elsewhere,
+                    True,
+                ],
+                [
+                    0,
+                    standard_deduction_if_dependent,
+                    deduction_amount,
+                ],
```

Comment:
instead of the last `True` case
```suggestion
                ],
            default = deduction_amount
```